test/e2e/tests/network/linea-discover-button.spec.ts

```diff
@@ -44,7 +59,7 @@ describe('Linea Network Discover Button', function (this: Suite) {
         await selectNetworkDialog.clickDiscoverButton();
 
         // Switch to the new tab that was opened
-        await driver.switchToWindowWithTitle('MetaMask Portfolio - Linea');
+        await driver.switchToWindowWithTitle('E2E Test Page');
```

Comment:
we change the window title as we are now mocking the page, but the assertion below for the expected url should continue to pass as it is